packages/ember-htmlbars/lib/keywords/unbound.js

```diff
@@ -1,8 +1,22 @@
 import Ember from 'ember-metal/core'; // assert
 
 /**
-@module ember
-@submodule ember-htmlbars
+  The `{{unbound}}` helper can be used with bound helper invocations to
+  render them in their unbound form.
+
+  ```handlebars
+  {{unbound (capitalize name)}}
+  ```
+
+  In the aforementioned example, if the `name` property changes, the helper
+  will not re-render.
+
+  @module ember
+  @submodule ember-htmlbars
```

Comment:
As in https://github.com/emberjs/ember.js/blob/master/packages/ember-routing-htmlbars/lib/keywords/action.js#L3 the submodule should be `ember-templates`